### PR TITLE
Humanize emails

### LINKED ISSUE #21233: Update email wordings
Update the default WooCommerce email wordings with new, improved ones.

Creating an issue around this so it doesn't slip through the cracks before 3.5 beta.

### PR DESCRIPTION
**Pull request by kloon**
### All Submissions:

* [x] Have you followed the [WooCommerce Contributing guideline](https://github.com/woocommerce/woocommerce/blob/master/.github/CONTRIBUTING.md)?
* [x] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?

### Changes proposed in this Pull Request:

This PR updates the copy of all the emails to make it more human and inviting to store admins and customers.

Closes #21233

### How to test the changes in this Pull Request:

1. Test all emails using HTML and plain text modes and ensure everything looks good and there are no grammer errors.
2. Admin emails to test: new order, cancelled order, failed order
3. Customer emails to test: on-hold, processing, completed, refunded both full and partially, invoice both with a pending and non-pending status, customer notes, password reset, new account.

### Other information:

* [x] Have you added an explanation of what your changes do and why you'd like us to include them?
* [ ] Have you written new tests for your changes, as applicable?
* [x] Have you successfully ran tests with your changes locally?

### Changelog entry

> Enhancement - Improve the default copy of all store emails.

### CODE REVIEW

File: includes/emails/class-wc-email-customer-completed-order.php [hunk 1]
```diff
@@ -90,13 +90,12 @@ public function get_default_subject() {
 		 * @return string
 		 */
 		public function get_default_heading() {
-			return __( 'Your order is complete', 'woocommerce' );
+			return __( 'Thanks for shipping with us', 'woocommerce' );
```
Comment (willgorham) [inline includes/emails/class-wc-email-customer-completed-order.php]: 'Shipping' -> 'Shopping'?

**Comment by kloon** [comment]
Screenshots of the different emails using all the defaults cc @warrendholmes @MarinaWoo 

Admin New Order - http://cld.wthms.co/XetMop
Admin Failed Order - http://cld.wthms.co/kKMV5A

Order On-Hold - http://cld.wthms.co/KVA8zZ
Processing Order - http://cld.wthms.co/X6qAtZ
Completed Order - http://cld.wthms.co/v51is2
Refunded Order - Partial http://cld.wthms.co/XIjN2T - Full http://cld.wthms.co/hwLnEF
Customer Invoice - Paid http://cld.wthms.co/sai6BU - Unpaid http://cld.wthms.co/XTCF5B
Customer Note - http://cld.wthms.co/wGxnWu
Reset Password - http://cld.wthms.co/0qDoeH
New Account - http://cld.wthms.co/0ZMtz1

**Comment by MarinaWoo** [comment]
Thanks for this @kloon -  will review on Monday.

File: templates/emails/customer-completed-order.php [hunk 2]
```diff
@@ -10,45 +10,52 @@
...
  * @hooked WC_Emails::email_header() Output the email header
  */
 do_action( 'woocommerce_email_header', $email_heading, $email ); ?>
 
-<p><?php printf( __( "Hi there. Your recent order on %s has been completed. Your order details are shown below for your reference:", 'woocommerce' ), get_option( 'blogname' ) ); ?></p>
-
+<?php /* translators: %s: Customer first name */ ?>
+<p><?php printf( __( 'Hi %s,', 'woocommerce' ), $order->get_billing_first_name() ); ?></p><?php // phpcs:ignore WordPress.XSS.EscapeOutput.OutputNotEscaped ?>
+<?php /* translators: %s: Site title */ ?>
+<p><?php printf( __( 'Your %s order has been marked complete on our side. We hope you enjoy it!', 'woocommerce' ), wp_specialchars_decode( get_option( 'blogname' ), ENT_QUOTES ) ); ?></p><?php // phpcs:ignore WordPress.XSS.EscapeOutput.OutputNotEscaped ?>
```
Comment (claudiulodro) [inline templates/emails/customer-completed-order.php]: The 'We hope you enjoy it!' message may be a bit tone deaf on certain types of products. I could see that making for some awkward emails . . .

**Review by claudiulodro — CHANGES_REQUESTED** [review]
Overall these phrasing changes look good, and I don't notice any code issues.  Just a couple questions/feedback around some of them. We can address them after beta is tagged if time does not permit this week.

File: includes/emails/class-wc-email-customer-processing-order.php [hunk 3]
```diff
@@ -56,7 +56,7 @@ public function __construct() {
 		 * @return string
 		 */
 		public function get_default_subject() {
-			return __( 'Your {site_title} order receipt from {order_date}', 'woocommerce' );
+			return __( 'Your {site_title} order is now complete', 'woocommerce' );
```
Comment (claudiulodro) [inline includes/emails/class-wc-email-customer-processing-order.php]: Noticed this in my testing. The order is 'processing' status not 'complete' status at this point.

**Comment by kloon** [comment]
@claudiulodro I will defer the questions to @MarinaWoo & @warrendholmes as they did the proposal, I implemented the changes as per the proposal using the same capitalisation and grammar as specified.

**Comment by MarinaWoo** [comment]
I'm reviewing. Tricky as the brief was to 'humanize' but actually, they need to be quite plain.

Noted on that point @claudiulodro!

Will submit a final batch of suggestions by end of day.

**Comment by MarinaWoo** [comment]
Thanks again for creating the preview, @kloon. Reflected a bit on @claudiulodro 's comment about tone and have removed some bits to make sure there is nothing jarring.

Please can you apply the following changes, then generate another batch of previews for me to do a final proof-read (and run by a second pair of eyes):

> Admin New Order - http://cld.wthms.co/XetMop

- Change the subject line to: '_[wc]: New order 3505_'
- Colon missing at the end of the first line: '_You've received the following order from Joe Shops:_' 
- In the final line, delete '_Woo!_' so it just reads: '_Over to you._'

> Admin Failed Order - http://cld.wthms.co/kKMV5A

- Change the subject line to: '_[wc]: Order 3505 has failed_'
- In the first line, delete '_Oh dear!_'.

> Admin Cancelled order - missing screenshot

- Change the subject line to: '_[wc]: Micheal Bluth has cancelled order 873_' (so it matches the formate of the other admin emails with the store name in []'s)
- Change the final line to: '_Thanks for reading._'

> Processing Order - http://cld.wthms.co/X6qAtZ

Checking: my understanding is that this triggers once payment has been received? If so:

- Change the subject line to: '_Payment received for your order_'
- Change email heading to: '_Thank you for your payment_'
- Change '_Woo!_' in the sign off to '_Thanks!_'

> Customer Invoice - Paid http://cld.wthms.co/sai6BU 

- Replace: '_Have a great day._' with '_Thanks for reading._'

> Customer Invoice - Unpaid http://cld.wthms.co/XTCF5B

- Replace: '_Have a great day._' with '_Thanks for reading._'

> Customer Note - http://cld.wthms.co/wGxnWu

- Replace: '_Have a great day._' with '_Thanks for reading._'

> Reset Password - http://cld.wthms.co/0qDoeH

- Change the first line to: '_Someone has requested a new password for the following account on wc:_'
- Change the second line to: '_If you didn't make this request, just ignore this email. If you'd like to proceed:_'
- Replace: '_Good luck!_' with '_Thanks for reading._'

One final thing: unsure about where there should be #'s in front of order number if the table if we don't add it to the subject lines etc? Probably should be consistent there, but I don't have strong feelings either way.

File: includes/emails/class-wc-email-customer-reset-password.php [hunk 4]
```diff
@@ -79,7 +79,7 @@ public function __construct() {
 		 * @return string
 		 */
 		public function get_default_subject() {
-			return __( 'Password reset for {site_title}', 'woocommerce' );
+			return __( 'Password Reset Request for {site_title}', 'woocommerce' );
```
[the remark this answers is not in the document]
Comment (kloon) [reply includes/emails/class-wc-email-customer-reset-password.php]: I believe this is because it describes the action.

**Comment by claudiulodro** [comment]
Latest changes look good. Merging this so it can be included in beta. If we need to make further changes let's do that as a new PR.

**Review by claudiulodro — APPROVED** [review]

**Pull request merged**